notebooks/logging-examples/logging-plots.ipynb

```diff
@@ -0,0 +1,318 @@
+       "      <td>0-1</td>\n",
+       "      <td>0.5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>NNE</td>\n",
+       "      <td>0-1</td>\n",
+       "      <td>0.6</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>NE</td>\n",
+       "      <td>0-1</td>\n",
+       "      <td>0.5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>ENE</td>\n",
+       "      <td>0-1</td>\n",
+       "      <td>0.4</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>E</td>\n",
+       "      <td>0-1</td>\n",
+       "      <td>0.4</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  direction strength  frequency\n",
+       "0         N      0-1        0.5\n",
+       "1       NNE      0-1        0.6\n",
+       "2        NE      0-1        0.5\n",
+       "3       ENE      0-1        0.4\n",
+       "4         E      0-1        0.4"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import plotly.express as px\n",
+    "from plotly import data\n",
+    "\n",
+    "df = data.wind()\n",
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "31e007bc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scatter_plot=px.scatter(df, x=\"direction\", y=\"frequency\", color=\"strength\")\n",
+    "#save our image for plotting\n",
+    "scatter_plot.write_image(\"saved-scatter-plot-for-logging.png\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "10688243",
+   "metadata": {},
+   "source": [
+    "![figure](saved-scatter-plot-for-logging.png)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "8544305b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bar_plot=px.bar(df, x=\"direction\", y=\"frequency\", color=\"strength\")\n",
+    "bar_plot_bytes=bar_plot.to_image(format=\"png\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "83347314",
+   "metadata": {},
+   "source": [
+    "![bar figure](saved-bar-plot-for-logging.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4a41ac4b",
+   "metadata": {},
+   "source": [
+    "## Logging your plot\n",
+    "In `rubicon_ml` `artifacts`are a catch-all for any other type of data that can be logged to a file. We can use artifacts to capture information that doesn't fall into the `experiment`, `dataframe`, `feature`, and  `parameter` category. \n",
+    "\n",
+    "Leveraging `artifacts`, plots can be logged just by passing its location in the `data_path` argument to `project.log_artifact()`. We can also log plots by passing its byte representation to the `data_bytes` argument to `project.log_artifact()` as well."
```

Comment:
this is good - just set it up with some more text earlier where I mentioned above